backend/tests/regression/answer_quality/cli_utils.py

```diff
@@ -0,0 +1,291 @@
+import json
+import os
+import platform
+import subprocess
+import sys
+import time
+from typing import Any
+
+from retry import retry
+
+
+def _run_command(command: str, timeout: int = 60) -> tuple[int | Any, str, str]:
+    process = subprocess.Popen(
+        command, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    )
+    try:
+        stdout, stderr = process.communicate(timeout=timeout)
+        return process.returncode, stdout.decode(), stderr.decode()
+    except subprocess.TimeoutExpired:
+        process.kill()
+        print(f"Command timed out after {timeout} seconds: {command}")
+        return -1, "", f"Command timed out after {timeout} seconds"
+
+
+def get_current_commit_sha() -> str:
+    print("Getting current commit SHA...")
+    returncode, stdout, stderr = _run_command("git rev-parse HEAD")
+
+    if returncode == 0:
+        sha = stdout.strip()
+        print(f"Current commit SHA: {sha}")
+        return sha
+    else:
+        print("Failed to get current commit SHA.")
+        print(f"Error: {stderr}")
+        sys.exit(1)
+
+
+def check_docker_running() -> int:
+    print("Checking if Docker is running...")
+    returncode, _, _ = _run_command("docker info")
+    return returncode == 0
+
+
+def start_docker() -> None:
```

Comment:
This is a nice touch but to be honest it's fine to just check if docker is running and just raise an exception and exit if it's not